spotbugs/etc/messages.xml

```diff
@@ -7025,6 +7033,29 @@ just use the constant. Methods detected are:
     ]]>
       </Details>
   </BugPattern>
+  <BugPattern type="PA_PUBLIC_ATTRIBUTES">
+    <ShortDescription>Field is both public and written by a method</ShortDescription>
+    <LongDescription>Field {1} is both public and written by a method</LongDescription>
+    <Details>
+   <![CDATA[
+    <p>
+    SEI CERT rule OBJ01-J requires that accessibility of fields must be limited:
+    https://wiki.sei.cmu.edu/confluence/display/java/OBJ01-J.+Limit+accessibility+of+fields
+    In general, requiring that no fields must be public is overkill and unrealistic. Even the rule
+    mentions that final fields may be public, except if the type of the field is a mutable reference
+    type which can be modified despite the reference itself being final. Beside final field, there
+    may be other usages for public fields: some public fields may serve as "flags" which affect the
+    behavior of the class. These fields are expected to be read by the containing class itself and
+    written by other classes. If a field is both written by the methods of the containing class
+    itself and from outside is suspicious. Consider to make these fields private and provide setters
```

Comment:
from outside is suspicious -> from outside, that is suspicious

Consider to make -> Consider making

Reply:
Both of these got fixed, so this conversation can be resolved.